src/components/NotesMindMapView/NotesMindMapViewComponent.js

```diff
@@ -11,21 +11,25 @@ import { StyledNotesMindMap } from 'components/NotesMindMapView/NotesMindMapView
 let visNetwork;
 
 // TODO: continue move to sagas
-export class NotesMindMapViewComponent extends Component {
+export class NotesMindMapViewComponent extends Component 
+{
```

Comment:
Please conform to the existing code style.

I have just added `"brace-style":  ["error", "1tbs", { "allowSingleLine": true }]` rule to `eslint` to enforce brace being on the same line in cases like this.
https://eslint.org/docs/rules/brace-style

Please merge master into your branch, run `npm run lint` and fix the issues.